integrations/langchain/pyproject.toml

```diff
@@ -1,14 +1,14 @@
 [tool.poetry]
 name = "skyvern-langchain"
-version = "0.1.6"
+version = "0.1.7"
 description = ""
 authors = ["lawyzheng <lawy@skyvern.com>"]
 packages = [{ include = "skyvern_langchain" }]
 readme = "README.md"
 
 [tool.poetry.dependencies]
 python = "^3.11,<3.12"
```

Comment:
once https://github.com/Skyvern-AI/skyvern/pull/2380 is merged, let's bump python version here as well.